File: Mathematics/fundamentals/chocolate_breakdown/C/0001_solution.c
/**
 * Solution of problem 'Chocolate Breakdown'.
 * @file 0001_solution.c
 * @author Nischay Sharma
 * @version 1.0
 * */

#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

int main() {
    
    int L;
    int P;
    scanf("%d %d ",&L,&P);
    while(L>P)
        L-=P;
    printf("%d ",L);
    return 0;
}

File: Mathematics/number_theory/even_sum/Python3/0001_solution.py
"""
 Solution of problem 'Even Sum'.
 @file 0001_solution.py
 @author Nischay Sharma
 @version 1.0
 """
num=int(input())

sum=0
for i in range(2,num+1):
    if i%2==0:
        sum+=i 
print(sum)